src/Extensions/Cabal.hs

```diff
@@ -14,12 +14,16 @@ module Extensions.Cabal
     , parseCabalFileExtensions
     , parseCabalExtensions
     , extractCabalExtensions
+
+      -- * Bridge between Cabal and GHC extensions
+    , cabalToGhcExtension
+    , toGhcExtension
```

Comment:
Decided to reexports those functions, so others won't need to write them by themselves anymore

Reply:
Very kind of you :fishing_pole_and_fish: :hugs: